test/e2e/serial/workload_placement_test.go

```diff
@@ -0,0 +1,134 @@
+		Expect(err).NotTo(HaveOccurred(), "cannot get %q scheduler from the cluster", nrosched.NROSchedObjectName)
+
+		schedulerName = nroSchedObj.Status.SchedulerName
+		Expect(schedulerName).NotTo(BeEmpty(), "cannot autodetect TAS name from the NRO Scheduler object")
+
+		By("Check worker nodes topology is ok to run the tests")
+		err = testFixture.Client.List(context.TODO(), &nodeResourceTopologyList)
+		Expect(err).NotTo(HaveOccurred(), "unable to find NodeResourceTopolyList in the cluster")
+
+		topoloyManagerPolicy := nrtv1alpha1.SingleNUMANodeContainerLevel
+		nodeResourceTopologies = e2enrt.FilterTopologyManagerPolicy(nodeResourceTopologyList.Items, topoloyManagerPolicy)
+		for _, nrt := range nodeResourceTopologies {
+			fmt.Printf("jlom NRT name %#v\n", nrt.Name)
+		}
+		if len(nodeResourceTopologies) == 0 {
+			Skip(fmt.Sprintf("cannot find at least one node with policy %q", string(topoloyManagerPolicy)))
+		}
+	})
+
+	AfterEach(func() {
+		err := e2efixture.Teardown(testFixture)
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	Context("Cluster with multiple worker nodes suitable", func() {
+
+		BeforeEach(func() {
+
+			By("Getting nodes from NodeGroups")
+			nrOperatorObj := &nropv1alpha1.NUMAResourcesOperator{}
+			nrOperatorObjKey := client.ObjectKey{Name: NROperatorObjectName}
+			err := testFixture.Client.Get(context.TODO(), nrOperatorObjKey, nrOperatorObj)
+			Expect(err).NotTo(HaveOccurred(), "cannot get %q operator from the cluster", NROperatorObjectName)
+
+			mcpList, err := machineconfigpools.GetNodeGroupsMCPs(context.TODO(), testFixture.Client, nrOperatorObj.Spec.NodeGroups)
+			Expect(err).NotTo(HaveOccurred(), "cannot get MachineConfigPolls from NRO %q", NROperatorObjectName)
+
+			var nodes []corev1.Node
+			for _, mcp := range mcpList {
+				nodeSelector, err := metav1.LabelSelectorAsSelector(mcp.Spec.NodeSelector)
+				Expect(err).NotTo(HaveOccurred(), "cannot convert MachineConfigPool %q NodeSelector into a label.Selector", mcp.Name)
+
+				nodeList := &corev1.NodeList{}
+				err = testFixture.Client.List(context.TODO(), nodeList, &client.ListOptions{LabelSelector: nodeSelector})
+				Expect(err).NotTo(HaveOccurred(), "cannot get NodeList from MachineConfigPooll %q using NodeSelector", mcp.Name)
+				nodes = append(nodes, nodeList.Items...)
+			}
+
+			for _, node := range nodes {
+				fmt.Printf("jlom Nodes: %#v\n", node.Name)
+			}
+
+			By("Checking: There should be 2+ nodes")
+			if len(nodes) < 2 {
+				Skip("There should be at least 2 nodes to run this test")
+			}
+
+			By("Checking: at least one node have +1 NUMA Zones")
+			atLeastOneNodeWithAtLeastTwoNUMAZones := false
+			for _, node := range nodes {
+				nrt, err := e2enrt.FindFromList(nodeResourceTopologies, node.Name)
+				Expect(err).NotTo(HaveOccurred())
+
+				if len(nrt.Zones) > 1 {
+					atLeastOneNodeWithAtLeastTwoNUMAZones = true
+					break
+				}
+			}
+			if !atLeastOneNodeWithAtLeastTwoNUMAZones {
```

Comment:
and skip here if the count is not enough (say < 2)

Reply:
ok